parser/parser.y

```diff
@@ -9103,6 +9104,20 @@ SelectStmtLimit:
 		$$ = &ast.Limit{Count: $3.(ast.ExprNode)}
 	}
 
+SelectStmtLimitSimple:
+	"LIMIT" Int64Num
```

Comment:
How about use `LengthNum` instead of `Int64Num`

Reply:
LengthNum is a good idea. Previously I thought limit can be negative integers, but I checked the MySQL manual and it says limit and offset must both be nonnegative integer constants, so I changed Int64Num to LengthNum.
https://dev.mysql.com/doc/refman/8.0/en/select.html
About why I didn't use LimitOption, LimitOption supports "?" format for PREPARE, but commands like "ADMIN SHOW DDL JOBS" and similar ones do not support this syntax, so I thought maybe just conform to them.